rules/manage-javascript-projects-with-nx/rule.md

```diff
@@ -0,0 +1,57 @@
+---
+type: rule
+archivedreason: 
+title: Tools - Do you use Nx to manage your huge JavaScript solution?
+guid: da45f247-e727-4f83-8a89-89cf2a407b31
+uri: manage-javascript-projects-with-nx
+created: 2023-02-10T01:26:45.262Z
+authors:
+- title: Chris Clement
+  url: https://www.ssw.com.au/people/chris-clement
+related: []
+
+---
+
+One of the main problems working on a huge monorepo solution is usually the development experience and the build time.
+[Nx](https://nx.dev/) is one of many tools that can improve this experience in JavaScript projects.
+
+<!--endintro-->
+
+The amount of code that needs to be processed by the compiler scales proportionally with the solution size. Hence, the compile time will grow naturally as the solution grows in size.
+This surely affects both the development experience and the team's velocity, leaving both developers and stakeholders unhappy.
+
+[Nx](https://nx.dev/) is a JavaScript build system that aims to make developing on monorepo solution easier and faster.
```

Comment:
@christoment no need to repeat the link
https://ssw.com.au/rules/do-you-avoid-redundant-linking-aka-single-link-to-single-location/

Reply:
Done :)